app/work/page.tsx

```diff
@@ -1,8 +1,188 @@
+      },
+      {
+        name: "JavaScript"
+      }
+    ],
+    image: '/assets/work/thumb1.png',
+    live: "",
+    github: "",
+  },
+  {
+    num: '02',
+    category: 'frontend',
+    title: 'project 2',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem odio placeat minima dolorem ad reiciendis aliquam, autem ut nisi, sunt similique suscipit nostrum expedita odit voluptatem accusantium id voluptas quod.',
+    stack: [
+      {
+        name:'Next.js'
+      },
+      {
+        name: "Tailwind.css"
+      },
+      {
+        name: "Node.js"
+      }
+    ],
+    image: '/assets/work/thumb2.png',
+    live: "",
+    github: "",
+  },
+  {
+    num: '03',
+    category: 'frontend',
+    title: 'project 3',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quidem odio placeat minima dolorem ad reiciendis aliquam, autem ut nisi, sunt similique suscipit nostrum expedita odit voluptatem accusantium id voluptas quod.',
+    stack: [
+      {
+        name:'Next.js'
+      },
+      {
+        name: "Tailwind.css"
+      },
+    ],
+    image: '/assets/work/thumb3.png',
+    live: "",
+    github: "",
+  },
+];
 
 const Work = () => {
+  
+  const [project, setProject ] = useState(projects[0]);
+
+  const handleSlideChange = (swiper: SwiperType) => {
+
+    const currentIndex = swiper.activeIndex;
+    setProject(projects[currentIndex]);
+  }
+
   return (
-    <div>Work</div>
+    <motion.div 
+      initial= {{opacity: 0}} 
+      animate={{opacity: 1, transition: {delay:2.4, duration: 0.4, ease: "easeIn"}}} 
+      className="min-h-[80vh] flex flex-col justify-center py-12 xl:px-0"
+    >
+      <div className="container mx-auto">
+        <div className="flex flex-col xl:flex-row xl:gap-[30px]">
+          <div className="w-full xl:w-[50%] xl:h-[460px] flex flex-col xl:justify-between order-2 xl:order-none">
+            <div className="flex flex-col gap-[30px] h-[50%]">
+              <div className="text-8xl leading-none font-extrabold text-transparent text-outline">
+                {project.num}
+              </div>
+              <h2 className="text-[42px] font-bold leading-none text-white group-hover:text-accent transition-all duration-500 capitalize">
+                {project.category} project
+              </h2>
+              <p className="text-white/60">{project.description}</p>
+              <ul>
+                {project.stack.map((item, index) => {
+                  return(
+                    <li key = {index} className="text-xl text-accent">
+                      {item.name}
+                      {index !== project.stack.length -1 && ","}
+                    </li>
+                  )
+                })}
+              </ul>
+              <div className="border border-white/20"></div>
+              <div className="flex items-center gap-4">
+                <Link href={project.live} target="_blank">
+                  <TooltipProvider delayDuration={100}>
```

Comment:
Can you add a screenshot of what this is in the UI? I don't quiet understand why there needs to be a provider here

Reply:
I referred this documentation -> https://ui.shadcn.com/docs/components/tooltip

Also see the screenshot attached. 
<img width="268" alt="Screenshot 2024-08-27 at 1 05 15 PM" src="https://github.com/user-attachments/assets/86d1b7fe-18ff-4a80-8852-d4e42eb4c588">

Reply:
I think you can use a single provider for both components